axolotl-web/src/store/store.js

```diff
@@ -6,7 +6,7 @@ Vue.use(Vuex)
 export default new Vuex.Store({
   state: {
     chatList: [],
-    messageList: [],
+    messageList: {},
```

Comment:
What about renaming `messageList` to `messages`?
Is the list part referring to the data type, or that the content is not a single item?

Follow-up question:
If the messageList object only contains the child Messages, could the structure be flattened?
`state.messageList.Messages`-> `state.Messages`

Reply:
The data structure contains everything the MessageList page needs to render, so it's like that
![image](https://user-images.githubusercontent.com/404058/94856835-d9908300-0430-11eb-928c-83c37960d699.png)

Reply:
Fantastic explanation, really like the screenshot.
What about leaving the `List` suffix for something which sounds less like a type, like `Items`?

Seeing the object messageList leads easily to make the assumption that it is a list, but in fact its an object containing a list.

Reply:
That's a good suggestion, but I'd like the refactor to be done in another PR, the main one about message status is becoming big enough.